src/frontend/types/libs/plyr/index.d.ts

```diff
@@ -1,35 +1,629 @@
-/* tslint:disable:no-namespace unified-signatures */
-// Type definitions for plyr 3.4.3
-// Project: https://github.com/sampotts/plyr
-// Definitions by: Mehdi Benadda https://github.com/mbenadda
+// Type definitions for plyr 3.3.15
```

Comment:
I'm not sure it's a great idea to put someone else's maybe-not-complete and maybe-not-up-to-date definition here.

Either it's good enough to be in `DefinitelyTyped` or we should probably just use the parts of the definition we need. Otherwise it's just bloat. WDYT @lunika @sampaccoud ?

Reply:
I have no problem with the mention, it's the wholesale copy/paste I'm not sure about